app/templates/events/view/sessions.hbs

```diff
@@ -0,0 +1,146 @@
+  <div class="search-box">
+    <div class="ui icon input">
+      <input class="prompt" type="text" placeholder="Search">
+      <i class="search icon"></i>
+    </div>
+    <div class="results"></div>
+  </div>
+  <br>
+  <br>
+  <div class="show enteries">
+    <div class="text-show"> Show </div>
+    <div class="left-float">
+      <select class="ui dropdown">
+        <option value="0">10</option>
+        <option value="1">20</option>
+        <option value="2">50</option>
+        <option value="3">100</option>
+      </select>
+    </div>
+    <div class="text-enteries"> Enteries </div>
+  </div>
+  <br>
+  <br>
+</div>
+<div class="row">
+  <div class="sixteen wide column">
+    <table class="ui fixed table">
+      <thead>
+        <tr>
+          <th>{{t 'State'}}</th>
+          <th>{{t 'Title'}}</th>
+          <th>{{t 'Speakers'}}</th>
+          <th>{{t 'Track'}}</th>
+          <th>{{t 'Short Abstract'}}</th>
+          <th>{{t 'Submission Date'}}</th>
+          <th>{{t 'Last Modified'}}</th>
+          <th>{{t 'Email Sent'}}</th>
+          <th></th>
+          <th></th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <td>
+            <div>
+              <button class="ui green button">Confirmed</button>
+            </div>
+          </td>
+          <td>
+            Test Session
+          </td>
+          <td>
+            <div class="ui ordered list">
+              <div class="item">Mario Behling</div>
+              <div class="item">Test</div>
+            </div>
+          </td>
+          <td>
+            Blockchain
+          </td>
+          <td>
+            Get an outlook of the awesome program of the weekend from track organizers and the content team.
+          </td>
+          <td>
+            March 18, 2017
+            <br> 09:30 AM
+          </td>
+          <td>
+            March 25,
+            <br> 09:30 PM
+          </td>
+          <td>
+            No
+          </td>
+          <td>
+            <div class="ui vertical compact basic buttons">
+              {{#ui-popup content=(t 'View session details') class='ui icon button'}}
+                <i class="unhide icon"></i>
+              {{/ui-popup}}
+              {{#ui-popup content=(t 'Edit session details') class='ui icon button'}}
+                <i class="edit icon"></i>
+              {{/ui-popup}}
+              {{#ui-popup content=(t 'Delete session') class='ui icon button'}}
+                <i class="trash outline icon"></i>
+              {{/ui-popup}}
+              {{#ui-popup content=(t 'Browse last 10 sessions') class='ui icon button'}}
+                <i class="browser icon"></i>
+              {{/ui-popup}}
+            </div>
+          </td>
+          <td>
+            <div class="ui vertical compact basic buttons" style="background:red">
+              {{#ui-popup content=(t 'Reject') class='ui icon button'}}
+                <i class="remove icon" style="color:white"></i>
```

Comment:
Again. `color:white`. If you had just used semantic-ui button classes, this would have been taken care of automatically. Please read the documentation.